internal/cmd/shim.go

```diff
@@ -101,12 +108,12 @@ func pluginWASM(p config.Plugin) *plugin.Codegen_WASM {
 	return nil
 }
 
-func pluginGoType(o config.Override) *plugin.ParsedGoType {
+func pluginGoType(o config.Override) *goopts.ParsedGoType {
```

Comment:
It's obviously not great to use the plugin-defined type here, since in a world where there's actual separation between sqlc and golang codegen we wouldn't be able to import it. But it seemed like the best choice for now.

Although maybe we can delegate all parsing of the go type to the plugin and not do any at sqlc config parse time?

Reply:
Ended up dropping this altogether after removing parse logic for go type in sqlc config